src/components/tabs/js/tabsController.js

```diff
@@ -558,17 +562,22 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
   }
 
   /**
+   * Calculates the width of the pagination wrapper by summing the widths of the dummy tabs.
    * @returns {number}
    */
   function calcPagingWidth () {
-    var width = 1;
+    return calcTabsWidth(getElements().dummies);
+  }
+
+  function calcTabsWidth(tabs) {
+    var width = 0;
```

Comment:
I believe this was to account for a wrapping bug regarding pagination where the last item was wrapping to a new line without it; however, this looks like a band-aid that was hastily thrown in rather than diagnosing the root cause.